pyk4a/config.py

```diff
@@ -74,7 +88,7 @@ def __init__(self,
         self.wired_sync_mode = wired_sync_mode
         self.subordinate_delay_off_master_usec = subordinate_delay_off_master_usec
         self.disable_streaming_indicator = disable_streaming_indicator
-        self.color_format = 3  # BGRA32
+        self.color_format = color_format
```

Comment:
This was harcoded because the rest of the code would crash if you set to something else. For exemple, the function [k4a_image_to_numpy](https://github.com/etiennedub/pyk4a/blob/master/pyk4a/pyk4a.cpp#L174) only support  BGRA and depth.

Reply:
will revert that back, I don't need the other modes for now and can't do the cpp changes easily (I'm more of a python guy).